src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div
+      class="Card"
+      data-qa="card"
+    >
+      <img
+        class="CardImage"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="CardTitle"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </h2>
+      <p class="CardCode">Product code: 195434</p>
+      <div class="CardRev">
+        <div class="Stars Stars--4">
+          <span class="StarsStar"></span>
+          <span class="StarsStar"></span>
+          <span class="StarsStar"></span>
+          <span class="StarsStar"></span>
+          <span class="StarsStar"></span>
+        </div>
```

Comment:
```suggestion
    <div
      class="card"
      data-qa="card"
    >
      <img
        class="card__image"
        src="./images/imac.jpeg"
        alt="imac"
      >
      <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </h2>
      <p class="card__code">Product code: 195434</p>
      <div class="card__review">
        <div class="stars stars--4">
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
        </div>
```
general convention for naming styles in css is kebab-case or some convention base on metodologies (for example BEM)